torch/lib/THC/generic/THCTensorMathReduce.cu

```diff
@@ -3,7 +3,7 @@
 #else
 
 THC_API void
-THCTensor_(sum)(THCState* state, THCTensor *self, THCTensor *src, long dimension, int keepdim) {
+THCTensor_(sum)(THCState* state, THCTensor *self, THCTensor *src, int dimension, int keepdim) {
```

Comment:
While I agree with this change, I'm commenting this here just to raise awareness to others of this change (the same repeats all over this file).